src/guide/contributing/translations.md

```diff
@@ -76,6 +77,9 @@ Devem ser traduzidos sempre da mesma maneira (a menos que sua tradução atrapal
 - _Props_ = Propriedades
 - _Hooks_ = Gatilhos
 - _Bind_ = Vínculo ou Interligação
+_ _Build time_ = Processo de construção
+_ _Release notes_ = Notas de Lançamento
+_ _Build_ = Distribuição
```

Comment:
Esse cara é controverso mesmo, acho que vale a pena traduzir mas deixar o original em inglês e itálico entre parênteses logo em seguida, assim como fazemos com "camada visual (_view layer_)" na introdução do guia.